pkg/syncer/cnsoperator/controller/cnsnodevmattachment/cnsnodevmattachment_controller.go

```diff
@@ -140,9 +143,40 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 	backOffDuration = make(map[string]time.Duration)
 
+	// Predicates are used to determine under which conditions
+	// the reconcile callback will be made for an instance.
+	pred := predicate.Funcs{
```

Comment:
Now that we are handling conflicts, do we still need the predicate?

Reply:
Removed the predicate in the latest commit.